tests/xtro-sharpie/Log.cs

```diff
@@ -11,9 +11,13 @@ static class Log {
 		public static IList<string> On (string fx)
 		{
 			List<string> list;
-			if (!lists.TryGetValue (fx, out list)) {
+
+			string existingName = lists.Keys.FirstOrDefault (x => String.Compare (x, fx, true) == 0);
```

Comment:
That won't stop dupes in the list so it could, in theory, use the wrong one (empty)

I think it might be easier to fix (and likely faster to run) if we use a `StringComparer.OrdinalIgnoreCase` when creating the `List<string>` instance.

Reply:
I think you might be misreading the diff, or I'm misreading the code.

We have one of these:

static Dictionary<string, List<string>> lists = new Dictionary<string, List<string>> ();

that the On method returns one of the stored lists. We previously were just using lists[fx] to look up directly by key. I changed it to walk the dictionary keys and find the one based on the case insensitive name. The bug was caused by two separate dictionary entries:

lists["PDFKit"] and lists["PdfKit" existing.

Reply:
My bad, it's the dictionary (not the list) that should have the equality comparer -> https://msdn.microsoft.com/en-us/library/ms132073(v=vs.110).aspx

Reply:
I didn't know you could do that with dictionaries... TIL

I'll redo with that.